src/components/PartnerStatsCard/index.jsx

```diff
@@ -0,0 +1,37 @@
+import React, { Component } from 'react';
+import Card from '../Dashboard/Card';
+import './partnersStatsCard.scss';
+
+class PartnerStats extends Component {
+  renderStatsCard = (type) => {
+    const renderDiv = value => (
```

Comment:
This is just a suggestion:  What if you made `renderDiv`  a stand-alone function by itself an call it in the `renderStartsCard` function to improve the readability of your code.